### ISSUE TITLE
Invalid Frame on verification of a BBS signature

### ISSUE THREAD

**Issue by vongohren** [mattrglobal/jsonld-signatures-bbs#122]
Im uncertain where this fails but seems to me that it comes from this lib.

I have a strane error coming from trying to recreate the bbs signature sample with libraries that I want to use.
I try to great a keypair with did-key.js and use that to sign.
That works
Then I want to verify but get a strange JSON LD verification error, and dont know where to start.

Any thoughts?

<img width="1680" alt="Screenshot 2021-05-06 at 17 03 15" src="https://user-images.githubusercontent.com/1012055/117321210-39e71780-ae8d-11eb-9778-7465d90647a1.png">

<img width="1680" alt="Screenshot 2021-05-06 at 17 03 30" src="https://user-images.githubusercontent.com/1012055/117321185-36539080-ae8d-11eb-88e4-b5496ef914b3.png">

**Comment by vongohren** [comment]
Some context

<img width="1484" alt="Screen Shot 2021-05-06 at 19 15 10" src="https://user-images.githubusercontent.com/1012055/117338999-83d8f900-ae9f-11eb-8822-9798a61a99ea.png">

**Comment by kdenhartog** [comment]
Can you publish the code that you're using to work with this? I'm thinking the issue here is that the verificationMethod used is only a fragment which means either an `@base` property needs to be specified within the issued credential or the full did url syntax should be added so that the documentLoader can resolve the public key material to verify the signatures.

**Comment by vongohren** [comment]
Thanks for reply @kdenhartog. Anywhere I can read about this @base property, or you know about any examples or helpers about that? There is btw a long thread going on here: https://difdn.slack.com/archives/C4X50SNUX/p1620313732057300, as it was relevant for some other questions I had up in the air.

I will see if your pointer and the discussion leads to something working tomorrow, dont have the code at hand now.

**Comment by vongohren** [comment]
@kdenhartog I do think I touched upon something bigger: https://github.com/w3c/did-imp-guide/pull/3, not directly related but there are some mentions of BBS and documents without context there.

**Comment by vongohren** [comment]
I was not able to get past this issue, even if I change up things in the document loader: https://gist.github.com/vongohren/92753c7dc8e8be7ed219c2d184c624aa

How shall I be able to verify this?

**Comment by vongohren** [comment]
I did get help from another DIF member: https://gist.github.com/S3bb1/ddd74704c0f9980d93e4e6302439594c
He left comments and thoughts on why it was needed.

But is this the stuff you have to deal with usually? Or since everyone is making their own things, they know their own libraries?

**Comment by tplooker** [comment]
@vongohren to be clear is this issue distinct from #114?

**Comment by vongohren** [comment]
@tplooker yeah thsi is different, as this is more related to the resolving. The other I believe is mainly related to the big gap in different versions for Json-ld signatures.
